core/src/main/scala/org/apache/spark/eventhubs/EventPosition.scala

```diff
@@ -81,7 +81,7 @@ object EventPosition {
    */
   def fromSequenceNumber(seqNo: SequenceNumber): EventPosition = {
     require(seqNo >= 0L, "Please pass a positive sequence number.")
-    EventPosition(seqNo = seqNo, isInclusive = true)
+    EventPosition(seqNo = seqNo)
```

Comment:
Could we remove the duplication?

Reply:
We have to keep this duplication, everything else is cleaned up 👍